### DISCUSSION TITLE
How do I promote a pane to a tab or how to make pane floating?

### DISCUSSION THREAD

**Discussion by mabasic** [zellij-org/zellij#2440] [Q&A] [answered]
I can't seem to find out how to make a pane full screen and then have it go back to its  previous position.

Use case:
- I have 4 horizontal panes  in a tab
- I want to make one page full size so that I can inspect its output better
- I want to be able to revert that pane back to its previous size (optional)

Is this possible to do?

**Comment by Lockszmith-GH** [comment, marked as answer]
`Ctrl`+`P` then `F`
It's called [Full Screen](https://zellij.dev/documentation/cli-actions.html?highlight=fullscreen#toggle-fullscreen).

Also check out [Swap Layouts](https://zellij.dev/documentation/swap-layouts.html) in the docs.

**Reply by mabasic** [reply]
Thank you very much.
